docker/build-debian.sh

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
+
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# Author: Dmitrii Demin
+#
+
+set -e
+
+OPS_DEPS_CORE=(
+  'libcurl3' \
+  'libssl1.1' \
+)
+
+OPS_DEPS_BOOST=(
+ 'libboost-filesystem' \
+ 'libboost-regex' \
+ 'libboost-thread' \
+)
+
+BUILD_DEPS=(
+ 'libboost-all-dev' \
+ 'libcurl4-openssl-dev' \
+ 'libgcrypt-dev' \
+ 'libgnutls28-dev' \
+ 'libsasl2-dev' \
+ 'libssl-dev' \
+ 'uuid-dev' \
+)
+
+BUILD_TOOLS=(
+ 'apt-transport-https' \
+ 'bzip2' \
+ 'ca-certificates' \
+ 'cmake' \
+ 'curl' \
+ 'dirmngr' \
+ 'g++' \
+ 'gcc' \
+ 'git' \
+ 'gnupg' \
+ 'make' \
+ 'scons' \
+)
+
+TEST_TOOLS=(
+ 'bc' \
+ 'nano' \
+ 'netcat' \
+ 'valgrind' \
+)
+
```

Comment:
For centos you have mongo as part of the test tools. Not necessary for debian?

Reply:
Debian uses another repository, there is a special block for it

Reply:
ok, great. Just checking ... :)